app/controllers/transcriptions_controller.rb

```diff
@@ -84,4 +84,10 @@ def allowed_filters
   def update_attr_whitelist
     ["flagged", "text", "status"]
   end
+
+  def jsonapi_serializer_params
+    {
+      serialize_text: url_options[:_recall][:action] == 'show'
```

Comment:
Use rails `action_name` method to get the controller action, nrails.org/action_controller_overview.html#routing-parameters
```suggestion
      serialize_text: action_name == 'show'
```

Reply:
Nice. Will update that.